### ISSUE TITLE
[BUG] Accessibility -  Web App -  Slider not reading properly to Screen reader

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5109]
**Describe the bug**
This bug is to address what is pronounced by the screen reader to the Mobile user on the Web app. 

**To Reproduce**
1. Go to 'app.illiniois.edu' on Chrome Browser of Android Mobile device while using TalkBack Screen reader
2. Click on 'Continue' through Orientation screens until Privacy screen
3. Swipe right 

**Expected behavior**
I would expect the swiping to stop at 5 since visually only 1- 5 is represented. 

**Actual behavior**
The users can swipe up until 100. 
When the user swipes all they hear is the number increasing. (No information about level of privacy is relayed.)

**Screenshots**

https://github.com/user-attachments/assets/3bca565d-d5bb-4204-9efd-dec92df7cd11

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome 
 - Version: 135.0.7049.85 (official build) (64-bit), 

**Smartphone (please complete the following information):**
Device: Google Pixel
OS: Android
Browser N/A this is an apk
Version 7.0.20 PROD

**Additional context**

**Comment by acovher2** [comment]
@pmarkhennessy @TodorBachvarov 
I do not think I should open a bug for every item missing screen reader labels. It would be redundant. Can we please make this the MAIN screen reader Accessibility EPIC ticket? Things like forms (create a group, editing profile, etc) all need form field labels and buttons need labels for screen reader users to be able to access this the same as others. 

Please let me know and I'll change the title and bug description to match. 

Thank you kindly,
Amanda

**Comment by acovher2** [comment]
Another example of this is on the Select and Image modal of the Group creation screens. 
See video with SR pronouncing odd labels. 

https://github.com/user-attachments/assets/171b29f5-f614-4309-9064-ef87dd30e692

**Comment by TodorBachvarov** [comment]
Hi @acovher2, I've made some changes in 7.3.20 that hopefully will fix the slider value announcement. This required a slight change in the announcement: instead of "Increased/Decreased to 4" now we set "Privacy level set to 4 of 5"
Note there was an issue for mobile where the user was hearing values 0 and 6 even when the slider was not moving to these values. This is also fixed.
